modules/render.js

```diff
@@ -1,12 +1,20 @@
 import Todo from './todo.js';
+import updateStatus from './todo-status.js';
 
 const todosList = document.querySelector('.todos-list');
 const todoInput = document.querySelector('.add-todo');
 const addBtn = document.querySelector('.add');
+const clearCompletedBtn = document.querySelector('.clear');
 let todosArr = [];
 
-const updateStatus = (index, bool) => {
-  todosArr[index].completed = bool;
+const updateIndex = (index) => {
+  for (let i = index + 1; i < todosArr.length; i += 1) {
+    todosArr[i].index -= 1;
+  }
+};
```

Comment:
✅ The updated array of items is stored in local storage, so the user gets the correct list values after the page reloads 👍🏻